modules/perception/camera/lib/obstacle/detector/yolo/region_output.cu

```diff
@@ -538,8 +603,8 @@ void get_objects_gpu(const YoloBlobs &yolo_blobs,
                   idx_sm,
                   stream);
     num_kept += indices[types[k]].size();
-    std::vector<float> conf_score(cpu_cls_data + k * num_candidates,
-                                  cpu_cls_data + (k + 1) * num_candidates);
+    std::vector<float> conf_score(cpu_cls_data + k * all_scales_num_candidates,
+                            cpu_cls_data + (k + 1) * all_scales_num_candidates);
```

Comment:
align indentation of all arguments in the next lanes to the first argument.
e.g.
std::vector<float> conf_score(
   . cpu_cls_data + k * all_scales_num_candidates,
     cpu_cls_data + (k + 1) * all_scales_num_candidates);